common/src/test/java/io/pravega/common/hash/HashHelperTest.java

```diff
@@ -50,5 +55,44 @@ public void testBytesToUUID() {
         assertEquals(uuid.getMostSignificantBits(), 0x0001020304050607L);
         assertEquals(uuid.getLeastSignificantBits(), 0x08090A0B0C0D0E0FL);
     }
+
+    @Test
+    public void testHashToBucketString() {
+        byte[] data = new byte[32];  // Short strings, otherwise this test would take too long.
+        testBucketUniformity(HashHelper::hashToBucket,
+                r -> {
+                    r.nextBytes(data);
+                    return new String(data);
```

Comment:
Should this use the randomString function? After all arbitrary arrays of bytes are not guaranteed to be valid strings.

Reply:
Oh yes, good catch. Fixed.